src/autorest-core/lib/autorest-core.ts

```diff
@@ -212,25 +212,22 @@ export class AutoRest extends EventEmitter {
         view.messageEmitter.removeAllListeners();
         return true;
       } catch (e) {
-        if (e instanceof Error) {
```

Comment:
just swapped this test since `Error`s marshalled back from safeEval fail this test. `Exception`s on the other hand are not thrown within safeEval so we can rely on that